File: BACKEND/SpringBootAPI/src/main/java/com/example/demo/entities/Booking.java
package com.example.demo.entities;

import jakarta.persistence.*;
import lombok.*;

import java.math.BigDecimal;
import java.time.LocalDateTime;
 
@Entity
@AllArgsConstructor
@NoArgsConstructor
@Getter
@Setter
@ToString
@Table(name = "booking")
public class Booking {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    @Column(name = "booking_id")
    private int bookingId;

    @Column(name = "booking_date")
    private String bookingDate;

    @Column(name = "no_of_seats")
    private int noOfSeats;

    @Column(name = "price")
    private double price;

    @Column(name = "total_cost")
    private double totalCost;
    


    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "user_id")
    private User user;

    @ManyToOne(fetch = FetchType.EAGER)
    @JoinColumn(name = "event_id")
    private Event event;
}

File: BACKEND/DOTNETAPI/Solution1/UserLoginService/Models/Booking.cs
using System;
using System.Collections.Generic;

namespace UserLoginService.Models;

public partial class Booking
{
    public int BookingId { get; set; }

    public DateTime BookingDate { get; set; }

    public int? NoOfSeats { get; set; }

    public decimal? Price { get; set; }

    public decimal? TotalCost { get; set; }

    public int UserId { get; set; }

    public int EventId { get; set; }

    public virtual Event Event { get; set; } = null!;

    public virtual ICollection<Payment> Payments { get; set; } = new List<Payment>();

    public virtual User User { get; set; } = null!;
}
